### PR TITLE
[Bugfix] Get the path to the modify file using the directory path and filename

### PR DESCRIPTION
**Pull request by hlts2**
As titled

## NOTE

The following is an example for dependecies update PR 
- https://github.com/vdaas/vald-client-node/pull/83

## Summary by CodeRabbit

- **Chores**
  - Updated file processing to include directory names in paths.
  - Disabled automated pull requests for dependency updates in the workflow.

### CODE REVIEW

File: .github/workflows/_update-deps.yaml [hunk 1]
```diff
@@ -44,14 +44,15 @@ jobs:
       - name: Set Git config
         run: |
           git config --global --add safe.directory ${GITHUB_WORKSPACE}
+      - uses: vdaas/vald-client-ci/.github/actions/dependabot@fix/add-directory-name
```
Comment (kpango) [inline .github/workflows/_update-deps.yaml:47]:
```suggestion
      - uses: vdaas/vald-client-ci/.github/actions/dependabot@main
```

Comment (kpango) [reply .github/workflows/_update-deps.yaml:47]: Others LGTM

Comment (hlts2) [reply .github/workflows/_update-deps.yaml:47]: Thank you for catching my mistake 🙏 
I apply your suggestion 👍

**Review by kpango — APPROVED** [review]

**Comment by hlts2** [comment]
@kpango Thank you for your review!
I will merge this PR 🚀

**Pull request merged**